openedx/core/djangoapps/user_api/verification_api/views.py

```diff
@@ -26,17 +26,25 @@ def get_serializer(self, *args, **kwargs):
         kwargs['context'] = self.get_serializer_context()
         if isinstance(instance, SoftwareSecurePhotoVerification):
             return SoftwareSecurePhotoVerificationSerializer(*args, **kwargs)
-        else:
```

Comment:
maybe we can create a dict mapping the model class to the serializer class so we don't have to do this if/elif block?

Reply:
The dict approach seems to become more unwieldy with instance.__class__.__name__ etc.